widgets/google-map/google-map.php

```diff
@@ -455,6 +478,14 @@ function get_template_variables( $instance, $args ) {
 
 		$styles = $this->get_styles( $instance );
 
+		$global_settings = $this->get_global_settings();
+		if ( empty( $instance['api_key_section']['api_key'] ) ) {
+			if ( ! empty( $global_settings['api_key'] ) ) {
+				$instance['api_key_section']['api_key'] = $global_settings['api_key'];
+			}
+		}
```

Comment:
This was removed in the develop branch, so it seems it was accidentally reintroduced in the merge.

Reply:
How does that look? Should I add something similar to `get_api_key` for the breakpoint?

Reply:
Did you make a change? It looks the same to me. Perhaps you still need to push.
Just to clarify, what I meant was this is supposed to be removed, as it's not being used anymore. I'm not sure how the API key relates to the breakpoint setting.